OxfordBodleian/Juel-Jensen/BDLaethf28.xml

```diff
@@ -133,13 +174,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
 
+                                <desc>At least two hands were involved in the writing of the manuscript, possibly more. 
+                                    The script changes repeatedly throughout, often within the same page (e.g. <locus target="#35 #90 #115"/>), 
+                                    leading to an overall impression of irregularity. All hands belong to the nineteenth–twentieth century.</desc>
+                                <date notBefore="1800" notAfter="1998"/>
+                           <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Holy names; elements of numerals and punctuation signs.</seg>
+
                             </handNote>
                         </handDesc>
 
                        <!-- <decoDesc>
 
                         </decoDesc>-->
 
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#38"/>
+                                    <desc type="findingAid">‘Nr. 19’ written in Amharic in the upper margin.</desc>
```

Comment:
Is "Nr." a normal translation of the Amharic? If so, I think we should use "No." instead for "number". Here and some lines below.